README.md

```diff
@@ -1,3 +1,150 @@
 # Progresso
```

Comment:
About the name: progress, progressbar and progressive are already taken on pypi (sign that this project should never exist? :) ).
Other idea: progressist.